nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -294,6 +294,11 @@ public void externalInitexternalInitSupportFields() {
 
   @Test
   public void generatedAsUnannotated() {
+    String generatedAnnot =
```

Comment:
`javax.annotation.Generated` has disappeared in JDK 11 🙁

Reply:
Is there an equivalent? What do people use instead?